custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -83,19 +83,29 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
                 return newValue;
             }, true);
 
+            vm.getPopupSubheading = function () {
```

Comment:
This report adds a subheading before the list of indicators, so I introduced that concept to the base class so that I could still use the `templatePopup` function